README.md

```diff
@@ -125,6 +126,14 @@ For more guidance on how to use RISC Zero, how RISC Zero projects are typically
 structured, and other resources useful to developers new to RISC Zero, see our
 [Getting Started page][quickstart].
 
+## Building from source
+
+You can also build from source instead downloading the latest release.
+You might want to do this if you are making changes to this code base, or want to try out a development version.
```

Comment:
I'd drop all this

Reply:
Dropped in https://github.com/risc0/risc0/pull/1723/commits/ca3c1415cf87a3ed66f767e3768f58d4b2e3c6d8